huaweicloud/services/waf/resource_huaweicloud_waf_domain.go

```diff
@@ -504,9 +505,26 @@ func resourceWafDomainCreate(ctx context.Context, d *schema.ResourceData, meta i
 		return diag.FromErr(err)
 	}
 
+	if d.Get("protect_status").(int) != protectStatusEnable {
```

Comment:
the value of `protectStatusEnable` is **1**, but the default value of `protect_status` is **0** in docs, please check it.